stores/sql/mysql/migrations/main/schema.sql

```diff
@@ -113,35 +75,40 @@ CREATE TABLE `contracts` (
   `id` bigint unsigned NOT NULL AUTO_INCREMENT,
   `created_at` datetime(3) DEFAULT NULL,
   `fcid` varbinary(32) NOT NULL,
-  `renewed_from` varbinary(32) DEFAULT NULL,
-  `contract_price` longtext,
-  `state` tinyint unsigned NOT NULL DEFAULT '0',
-  `total_cost` longtext,
+  `host_key` varbinary(32) NOT NULL,
+
+  `archival_reason` varchar(191) NOT NULL DEFAULT '',
   `proof_height` bigint unsigned DEFAULT '0',
+  `renewed_from` varbinary(32) DEFAULT NULL,
+  `renewed_to` varbinary(32) DEFAULT NULL,
   `revision_height` bigint unsigned DEFAULT '0',
   `revision_number` varchar(191) NOT NULL DEFAULT '0',
   `size` bigint unsigned DEFAULT NULL,
   `start_height` bigint unsigned NOT NULL,
+  `state` tinyint unsigned NOT NULL DEFAULT '0',
   `window_start` bigint unsigned NOT NULL DEFAULT '0',
   `window_end` bigint unsigned NOT NULL DEFAULT '0',
-  `upload_spending` longtext,
-  `download_spending` longtext,
-  `fund_account_spending` longtext,
+
+  `contract_price` longtext,
+  `initial_renter_funds` longtext,
+
   `delete_spending` longtext,
-  `list_spending` longtext,
-  `host_id` bigint unsigned DEFAULT NULL,
+  `fund_account_spending` longtext,
+  `sector_roots_spending` longtext,
+  `upload_spending` longtext,
   PRIMARY KEY (`id`),
   UNIQUE KEY `fcid` (`fcid`),
-  KEY `idx_contracts_window_end` (`window_end`),
-  KEY `idx_contracts_host_id` (`host_id`),
-  KEY `idx_contracts_renewed_from` (`renewed_from`),
-  KEY `idx_contracts_state` (`state`),
+  KEY `idx_contracts_archival_reason` (`archival_reason`),
+  KEY `idx_contracts_fcid` (`fcid`),
+  KEY `idx_contracts_host_key` (`host_key`),
   KEY `idx_contracts_proof_height` (`proof_height`),
-  KEY `idx_contracts_start_height` (`start_height`),
-  KEY `idx_contracts_fc_id` (`fcid`),
+  KEY `idx_contracts_renewed_from` (`renewed_from`),
+  KEY `idx_contracts_renewed_to` (`renewed_to`),
   KEY `idx_contracts_revision_height` (`revision_height`),
+  KEY `idx_contracts_start_height` (`start_height`),
+  KEY `idx_contracts_state` (`state`),
   KEY `idx_contracts_window_start` (`window_start`),
-  CONSTRAINT `fk_contracts_host` FOREIGN KEY (`host_id`) REFERENCES `hosts` (`id`)
```

Comment:
Do we need to get rid of the constraint? We could also set `hostd_id` to `NULL` when we archive the contract.
That way we have strong guarantees around not deleting hosts for active contracts.